app.py

```diff
@@ -163,11 +163,191 @@ def signUp():
     del result['access_token']
     del result['verification_token']
 
+    db.session.close()
+
     # Return the user
     return {
-        "user": result
+        "user": session['access_token']
```

Comment:
Wait why was this changed to `session['access_token]`? We wouldn't wanna return the access_token to the user.